README.MD

```diff
@@ -14,6 +14,23 @@ for comparison are included. The comparisons are based on scoring
 philosophies for three statistical data types including categorical,
 continuous, and probabilistic.
 
+Possible enhancements include the ability to trace attributes from the
+candidate and benchmark maps to the outputs for analysis purposes.
+Additionally, we plan to enable comparisons across a hierarchy of
+samples beyond the current map level samples. These sample levels
+include catalogs (possibly using [STAC](https://stacspec.org/en)) and
+sub-samples (zonal level metrics within maps).
+
+Completed functionality:
+
+- Homogenize candidate and benchmark rasters for comparisons.
+- Two-class and multi-class categorical comparison case for 2 or
+    3-dimensional rasters.
+
+In-progress:
+
+- Documentation, Github actions, and continuous statistic types.
```

Comment:
Can you update this? You can move the Documentation and GH actions to completed as well.

```
In-progress:
- Refactoring public API for logic and readability.
- Adding examples.
- Memory performance improvements.
```

Reply:
I had not intended to add this portion back, I eliminated this part of the README previously.